packages/rest/src/utils/CredentialUtils.ts

```diff
@@ -0,0 +1,36 @@
+import type { Agent } from "@aries-framework/core";
+
+import { CredentialRepository } from "@aries-framework/core";
+import { IndyHolderService } from "@aries-framework/core/build/modules/indy/services/IndyHolderService";
+
+export class CredentialUtils {
+  private agent: Agent;
+  private credentialRepository: CredentialRepository;
+  private holderService: IndyHolderService;
+
+  public constructor(agent: Agent) {
+    this.agent = agent;
+    this.credentialRepository = this.agent.injectionContainer.resolve(CredentialRepository);
```

Comment:
I think most methods here that rely on the credential repository can now be directly accesed under `agent.credentials.XXX()`